sdk/core/Azure.Core/tests/TestFramework/RecordedTestSanitizer.cs

```diff
@@ -39,10 +39,7 @@ public virtual byte[] SanitizeBody(string contentType, byte[] body)
             return body;
         }
 
-        public virtual void SanitizeConnectionString(ConnectionString connectionString)
-        {
-        }
-
+        public virtual string SanitizeConnectionString(string connectionString) => connectionString;
```

Comment:
Similar to comment above, let's try to avoid changing this method signature unless there's a super compelling reason to do this.  It looks like it's also used in identity and storage, so we'd need to update those libraries as well if we changed this.